examples/spark2/src/main/scala/org/apache/carbondata/examples/AlterTableExample.scala

```diff
@@ -55,17 +59,17 @@ object AlterTableExample {
     spark.sql(
       s"""
          | CREATE TABLE carbon_table(
-         |    shortField short,
-         |    intField int,
-         |    bigintField long,
-         |    doubleField double,
-         |    stringField string,
-         |    timestampField timestamp,
-         |    decimalField decimal(18,2),
-         |    dateField date,
-         |    charField char(5),
-         |    floatField float,
-         |    complexData array<string>
+         | shortField short,
```

Comment:
if you are going to change this, please change all keyword to capital letter, like short to SHORT